capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```
@@ -21,65 +24,26 @@ export class WikiMediaComponent implements OnInit, OnDestroy {
   constructor(private wikiService: WikiResultsService) { }
 
   ngOnInit(): void {
-    this.getResults('New York City');
+    this.getResults('New York City, NY');
   }
 
   ngOnDestroy(): void {
     this.destroy$.next();
   }
 
   getResults(queryString: string) {
-    // TODO(sarahhud): Add error handling for wikipedia request and testing.
     this.wikiService.search(queryString)
-      .pipe(takeUntil(this.destroy$))
-      .subscribe((result: WikiSearchResult) => {
-        this.wikiResult = result;
-        if (this.wikiResult.parse.text) {
-          this.body = this.fixString(result.parse.text['*']);
+      .subscribe((result: WikiServiceResult) => {
+        if (result.history) {
+          this.body = result.history;
+          this.history = result.history;
+          this.urls = result.furtherReading
+          this.title = result.title;
           this.loading = false;
         }
         else {
-          console.log('API did not return a valid response.');
+          this.error = 'API did not return a valid response.';
```

also should raise the error in the console (console.error)